### PR TITLE
Use only necessary files for installation in Dockerfile

### PR DESCRIPTION
**Pull request by yardenshoham**
Reduced 456MB to 449MB.

Inspired by the discussion from #148. Can we make it even smaller? What does `cmake --install build --config Release` actually do?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,30 +20,41 @@ RUN git submodule update --init --recursive \
...
+COPY --from=builder lib lib
 
 # The docker-entrypoint.sh file
 COPY --from=permissions-giver /out/docker-entrypoint.sh /out/docker/docker-entrypoint.sh
 
-FROM alpine:3.12 AS builder
+FROM alpine:3.12 AS runner
 
 # Install cmake
 RUN apk --no-cache add cmake clang clang-dev make gcc g++ libc-dev linux-headers
```
Comment (SuperFola) [inline Dockerfile:53]: I'm not sure but I think we just need cmake and maybe make to have this working, the whole compiler thing could maybe be avoided

**Comment by SuperFola** [comment]
Nice, thanks!

I think we can make it smaller yep.

`cmake --install build --config Release` is literally installing ArkScript in the system, copying `Ark` into `/usr/local/bin`, and the lib folder into `/usr/local/lib`

File: Dockerfile (hunk 1 above)
Comment (yardenshoham) [reply Dockerfile:53]: You're correct!

**Comment by yardenshoham** [comment]
Reduced to 62.7MB! Awesome!

**Comment by yardenshoham** [comment]
> Nice, thanks!
> 
> I think we can make it smaller yep.
> 
> `cmake --install build --config Release` is literally installing ArkScript in the system, copying `Ark` into `/usr/local/bin`, and the lib folder into `/usr/local/lib`

I tried to change runner to

```Dockerfile
FROM alpine:3.12 AS runner

# Install Ark
COPY --from=organizer /out/ark/build/Ark /usr/local/bin/Ark
COPY --from=organizer /out/ark/lib /usr/local/lib
COPY --from=organizer /out/docker /usr/local/bin
ENTRYPOINT [ "docker-entrypoint.sh" ]
CMD [ "Ark" ]
```

But running I get this:

```bash
yarden@DESKTOP-R6TFLMI:~$ docker run -it --rm ark --help
Error loading shared library libstdc++.so.6: No such file or directory (needed by /usr/local/bin/Ark)
Error loading shared library libgcc_s.so.1: No such file or directory (needed by /usr/local/bin/Ark)
Error relocating /usr/local/bin/Ark: _ZNSo9_M_insertImEERSoT_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt9basic_iosIcSt11char_traitsIcEE4initEPSt15basic_streambufIcS1_E: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7reserveEm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem7__cxx1118directory_iteratorC1ERKNS0_4pathENS_17directory_optionsEPSt10error_code: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt5ctypeIcE13_M_widen_initEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE14_M_replace_auxEmmmc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo5tellpEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt4endlIcSt11char_traitsIcEERSt13basic_ostreamIT_T0_ES6_: symbol not found
Error relocating /usr/local/bin/Ark: _ZdlPv: symbol not found
Error relocating /usr/local/bin/Ark: _ZnwmRKSt9nothrow_t: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt18_Rb_tree_decrementPSt18_Rb_tree_node_base: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE5rfindEcm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8ios_baseD2Ev: symbol not found
Error relocating /usr/local/bin/Ark: _Znam: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem7__cxx1118directory_iteratorppEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt18_Rb_tree_incrementPSt18_Rb_tree_node_base: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE5eraseEmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem7__cxx114path5_ListC1ERKS2_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1115basic_stringbufIcSt11char_traitsIcESaIcEE7_M_syncEPcmm: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_end_catch: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSi4readEPcl: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt14basic_ofstreamIcSt11char_traitsIcEED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_allocate_exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE4findEcm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE17find_first_not_ofEPKcmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8ios_baseC2Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEPKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSi5tellgEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt6localeC1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem12current_pathB5cxx11Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZStlsISt11char_traitsIcEERSt13basic_ostreamIcT_ES5_PKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt8__detail20_Prime_rehash_policy14_M_need_rehashEmmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13basic_filebufIcSt11char_traitsIcEE4openEPKcSt13_Ios_Openmode: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1119basic_istringstreamIcSt11char_traitsIcESaIcEED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo9_M_insertIdEERSoT_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt14basic_ifstreamIcSt11char_traitsIcEED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt7getlineIcSt11char_traitsIcESaIcEERSt13basic_istreamIT_T0_ES7_RNSt7__cxx1112basic_stringIS4_S5_T1_EES4_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt10filesystem7__cxx114path5_List13_Impl_deleterclEPNS2_5_ImplE: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_guard_release: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem6statusERKNS_7__cxx114pathE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE12_M_constructEmc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem18create_directoriesERKNS_7__cxx114pathE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo5writeEPKcl: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt10filesystem7__cxx114path5_List3endEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt19__throw_regex_errorNSt15regex_constants10error_typeE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo5flushEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt9basic_iosIcSt11char_traitsIcEE5widenEc: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt20__throw_out_of_rangePKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE8_M_eraseEmm: symbol not foundError relocating /usr/local/bin/Ark: _ZNKSt7__cxx1115basic_stringbufIcSt11char_traitsIcESaIcEE3strEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo3putEc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem7__cxx114pathdVERKS1_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo9_M_insertIPKvEERSoT_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_assignERKS4_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt12__basic_fileIcE7is_openEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13basic_filebufIcSt11char_traitsIcEE5closeEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13runtime_errorC2EPKc: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_bad_cast: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13basic_filebufIcSt11char_traitsIcEED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE10_M_replaceEmmPKcm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem7__cxx114path14_M_split_cmptsEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt25__throw_bad_function_callv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo9_M_insertIyEERSoT_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSo9_M_insertIlEERSoT_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt10filesystem7__cxx114path17_M_find_extensionEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt10filesystem7__cxx1118directory_iteratordeEv: symbol not found
Error relocating /usr/local/bin/Ark: __dynamic_cast: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_mutateEmmPKcm: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt16__throw_bad_castv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE13find_first_ofEPKcmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8__detail15_List_node_base9_M_unhookEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem10remove_allERKNS_7__cxx114pathE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13runtime_errorC2ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSi10_M_extractIlEERSiRT_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_appendEPKcm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem7__cxx114path5_ListC1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZdaPv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE16find_last_not_ofEPKcmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt9use_facetISt5ctypeIcEERKT_RKSt6locale: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13runtime_errorC1EPKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt6localeC1ERKS_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt10filesystem7__cxx114path11parent_pathEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt9basic_iosIcSt11char_traitsIcEE5clearESt12_Ios_Iostate: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE4findEPKcmm: symbol not foundError relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9push_backEc: symbol not foundError relocating /usr/local/bin/Ark: __cxa_begin_catch: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt6locale2id5_M_idEv: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_rethrow: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_throw: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt6localeD1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem15last_write_timeERKNS_7__cxx114pathE: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt20__throw_system_errori: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8ios_base4InitC1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSi5seekgElSt12_Ios_Seekdir: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt9exceptionD2Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEaSEOS4_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8ios_base6xallocEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13runtime_errorC1ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13basic_filebufIcSt11char_traitsIcEEC1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _Znwm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt12__basic_fileIcED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt11_Hash_bytesPKvmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6substrEmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt9use_facetINSt7__cxx117collateIcEEERKT_RKSt6locale: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8__detail15_List_node_base11_M_transferEPS0_S1_: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt24__throw_out_of_range_fmtPKcz: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt6chrono3_V212system_clock3nowEv: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_guard_abort: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8__detail15_List_node_base7_M_hookEPS0_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_createERmm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEEC1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZdlPvm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem8relativeERKNS_7__cxx114pathES3_: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt24__throw_invalid_argumentPKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7replaceEmmPKcm: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8ios_base13_M_grow_wordsEib: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt20__throw_length_errorPKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEC1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEmmRKS4_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSolsEi: symbol not found
Error relocating /usr/local/bin/Ark: _Unwind_Resume: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt9terminatev: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt29_Rb_tree_insert_and_rebalancebPSt18_Rb_tree_node_baseS0_RS_: symbol not foundError relocating /usr/local/bin/Ark: _ZNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEED1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt17__throw_bad_allocv: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt6localeaSERKS_: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt3_V215system_categoryEv: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt16__ostream_insertIcSt11char_traitsIcEERSt13basic_ostreamIT_T0_ES6_PKS3_l: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_free_exception: symbol not found
Error relocating /usr/local/bin/Ark: __cxa_guard_acquire: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt10filesystem16create_directoryERKNS_7__cxx114pathE: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt19__throw_logic_errorPKc: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv120__function_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVN10__cxxabiv119__pointer_type_infoE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt9exception: symbol not found
Error relocating /usr/local/bin/Ark: _ZTTSt14basic_ofstreamIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt7nothrow: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt8ios_base4InitD1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt13runtime_error: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt12system_error: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt13runtime_errorD1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt14basic_ifstreamIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt11regex_errorD1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt12system_errorD1Ev: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt14basic_ofstreamIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt3cin: symbol not found
Error relocating /usr/local/bin/Ark: _ZTTSt14basic_ifstreamIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt12system_error: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt12system_error: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt13basic_filebufIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: __gxx_personality_v0: symbol not found
Error relocating /usr/local/bin/Ark: _ZTTNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt9basic_iosIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt15basic_streambufIcSt11char_traitsIcEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTTNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt11regex_error: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVNSt7__cxx1115basic_stringbufIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTISt5ctypeIcE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVNSt7__cxx1119basic_istringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTINSt6locale5facetE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTTNSt7__cxx1119basic_istringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
Error relocating /usr/local/bin/Ark: _ZTVSt11regex_error: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt4cout: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt4cerr: symbol not found
Error relocating /usr/local/bin/Ark: _ZSt4clog: symbol not found
Error relocating /usr/local/bin/Ark: _ZNSt5ctypeIcE2idE: symbol not found
yarden@DESKTOP-R6TFLMI:~$
```

Any idea what else `cmake --install build --config Release` is doing?

**Comment by SuperFola** [comment]
Hum I don't have this issue but another one one my version, pretty similar but much shorter. That's related to things we didn't install, the lib c++ shipped with one of the compiler, though we don't need the dev version only the .so files

I'll try to understand that as well

**Comment by SuperFola** [comment]
Maybe I don't understand docker well yet, but apparently everything is fine for me with the current version, I had to add the `-it` parameter to the command:
```
23:41 $ sudo docker run -it arkscript/nightly
ArkScript REPL -- Version 3.0.11 [LICENSE: Mozilla Public License 2.0]
Type "(quit)" to quit.
main:001> (print "hello world")
hello world
main:002> (let a 12)
main:003> (print (+ a 12))
24
main:004> (quit)
✔ ~/git/Ark/examples [dev ↓·107|…2]
23:42 $ sudo docker run arkscript/nightly
ArkScript REPL -- Version 3.0.11 [LICENSE: Mozilla Public License 2.0]
Type "(quit)" to quit.
terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_M_construct null not valid
```

just a problem I got, I can't mount file on my host to the container to execute them, or maybe I'm doing it wrong?
```
✘-125 ~/git/Ark/examples [dev ↓·107|…2]
23:38 $ sudo docker run -it -v $(pwd)/church-encoding.ark:/out/ark/c.ark arkscript/nightly /out/ark/c.ark
/usr/local/bin/docker-entrypoint.sh: exec: line 8: /out/ark/c.ark: Permission denied
✘-126 ~/git/Ark/examples [dev ↓·107|…2]
23:39 $ sudo docker run -it -v $(pwd)/church-encoding.ark:/out/docker/c.ark arkscript/nightly /out/docker/c.ark
/usr/local/bin/docker-entrypoint.sh: exec: line 8: /out/docker/c.ark: Permission denied
✘-126 ~/git/Ark/examples [dev ↓·107|…2]
23:40 $ sudo docker run -it -v $(pwd)/church-encoding.ark:/usr/c.ark arkscript/nightly /usr/c.ark
/usr/local/bin/docker-entrypoint.sh: exec: line 8: /usr/c.ark: Permission denied
✘-126 ~/git/Ark/examples [dev ↓·107|…2]
23:40 $ sudo docker run -it -v $(pwd)/church-encoding.ark:/usr/local/bin/c.ark arkscript/nightly /usr/local/bin/c.ark
/usr/local/bin/docker-entrypoint.sh: exec: line 8: /usr/local/bin/c.ark: Permission denied
```

**Comment by yardenshoham** [comment]
That happened to me too. I wonder why, we'll have to research how other Docker images handle permissions. I took inspiration from the `node` Docker image.

**Pull request merged**